src/data/relics.js

```diff
@@ -295,9 +319,141 @@ export const RELICS = [
       ],
       icon: images['relic-equilibrium.png'],
     },
+    {
```

Comment:
Added `tier` and `id` to each relic - then added all the t1-3 relics to the t4 array item.
There's a check in the RelicUnlockModal now to only render relics in a tier that are of that tier (t4) but that check is removed when displaying reloaded relics